README.md

```diff
@@ -6,13 +6,25 @@ This operator is installed by the [cluster-storage-operator](https://github.com/
 
```

Comment:
Please add a note that authoritative YAML files to deploy the driver are in github.com/openshift/cluster-storage-operator/tree/master/assets/csidriveroperators/aws-ebs

Reply:
I still don't see the note in the PR. The rest of the PR looks good to me.

Reply:
done 👍🏽  and sorry for the delay

Reply:
BTW, after reviewing Mike's PR [here](https://github.com/openshift/openstack-cinder-csi-driver-operator/pull/45) I noticed that the Dockerfiles have a reference to the `manifests/` directory. Please remove that too.